airbyte-webapp/src/components/hooks/services/useDbMigrationHook.tsx

```diff
@@ -0,0 +1,32 @@
+import DbMigrationResource, {
+  DbMigrationInfoResult,
+  DbMigrationMigrateResult,
+} from "core/resources/DbMigration";
+import { useAnalytics } from "../useAnalytics";
+import { useFetcher } from "rest-hooks";
+
+type DbMigrationService = {
+  info: (database: string) => Promise<DbMigrationInfoResult>;
+  migrate: (database: string) => Promise<DbMigrationMigrateResult>;
+};
+
+const useDbMigration = (): DbMigrationService => {
+  const analyticsService = useAnalytics();
+  const getMigrationInfo = useFetcher(DbMigrationResource.infoShape());
```

Comment:
as described later - get requests ( which in our cases are list and get that we send with post ) in most of the cases should be used with `useResource`. `useFetcher` is only for the manual call, that we actually don't need for this case.

as a solution you could declare getMigrationInfo here as 

`() => useResource(DbMigrationResource.infoShape())` and use later as

`function Component(){
   const { info } = useDbMigration();
   const migrations = info();
}